### ISSUE TITLE
Powercell: Available power doubles upon placing second powercell.

### ISSUE THREAD

**Issue by Undeadbanana** [McJty/RFTools#1585]
Unsure if it triples, quadruples, etc as I'm poor at the moment.

![2018-01-26_17 12 43](https://user-images.githubusercontent.com/4379427/35467604-9f616bc8-02c5-11e8-9b8d-f804ae6ecbcd.png)
![2018-01-26_17 12 50](https://user-images.githubusercontent.com/4379427/35467605-9f778bd8-02c5-11e8-9e3d-5b57ef994a52.png)
![2018-01-26_17 12 54](https://user-images.githubusercontent.com/4379427/35467606-9f8e6740-02c5-11e8-88f0-f9735776db2a.png)

**Comment by McJty** [comment]
Was that a powercell that was freshly created or did it contain a power card to link it? Also remember that when you have a powercell that is part of a powercell network and you break it it will retain some of its power and give that back if you place it back

**Comment by Undeadbanana** [comment]
Made two powercells, initially linking was acting weird so I relogged just in case it was a clientside thing. Relogging cleared it and linking the power cells together worked. 

That's probably it though, retaining the energy. I had one Inputting and the other outputing energy. When I upgraded them I had around 800 KRF, so I thought I messed up by upgrading them while charged since they started back from zero but then the power jumped to 300KRF. I'm pretty low tech when it comes to energy and I had a builder mining, so these guys weren't charging up very fast. That's why I thought maybe it was bugged. Both of these were just upgraded from p revious level
